.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+    "search.exclude": {
+        "**/vendor": true,
```

Comment:
Not neccassarily - I personally have configured vscode to show `.git` because I want to edit files like `.git/info/exclude`